x/axelarnet/module_test.go

```diff
@@ -75,8 +75,8 @@ func TestGetMigrationHandler(t *testing.T) {
 			},
 		}
 
-		transferK := ibctransferkeeper.NewKeeper(encCfg.Codec, sdk.NewKVStoreKey("transfer"), transferSubspace, &mock.ChannelKeeperMock{}, &mock.PortKeeperMock{}, accountK, bankK, scopedTransferK)
-		appModule = axelarnet.NewAppModule(k, &mock.NexusMock{}, bankK, accountK, ibcK, ibctransfer.NewAppModule(transferK), log.TestingLogger())
+		transferK := ibctransferkeeper.NewKeeper(encCfg.Codec, sdk.NewKVStoreKey("transfer"), transferSubspace, &mock.ChannelKeeperMock{}, &mock.ChannelKeeperMock{}, &mock.PortKeeperMock{}, accountK, bankK, scopedTransferK)
```

Comment:
why the channel keeper duplication?

Reply:
See [this](https://github.com/axelarnetwork/axelar-core/pull/1500#discussion_r888012178) from #1500